JS/Lesson22/sinan/index.html

```diff
@@ -1,11 +1,51 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en"></html>
```

Comment:
```suggestion
<html lang="en">
```
I guess the autocomplete closed the html tag for you by a mistake. Please fix :)